### DISCUSSION TITLE
Translate elements in attachment element

### DISCUSSION THREAD

**Discussion by natera** [Budibase/budibase#8248] [Help] [answered]
Hi, I need to translate the heading and link info in the attachment element, but I can't figure it out I tried with custom css but i dont know css :face_with_head_bandage: 

.spectrum-Link{
    visibility: hidden;
    position: relative;
}
.spectrum-Link:after {
    visibility: visible;
    position: absolute;
    top: 0;
    left: 0;
    content: "This text replaces the original.";
}

![imagen](https://user-images.githubusercontent.com/4206506/195655650-d11bf83a-1928-4fd9-8ed6-ded51a838b39.png)

**Comment by melohagan** [comment, marked as answer]
Hey @natera 

Your CSS works! You just need to put it inside style tags:

```html
<style>
.spectrum-Link {
visibility: hidden;
position: relative;
}
.spectrum-Link:after {
visibility: visible;
position: absolute;
top: 0;
left: 0;
content: "Replace the original.";
}
</style>
```

![Screenshot 2022-10-14 at 14 24 02](https://user-images.githubusercontent.com/101575380/195857878-467cf392-52e0-4b8c-a480-a7a95a4cda5e.png)

**Reply by natera** [reply]
Thank you so much @melohagan
